### DISCUSSION TITLE
How to change style without styler function ?

### DISCUSSION THREAD

**Discussion by heejit** [cogentcore/core#1236] [General]
I want to update the Widget style before and after function call ?

**Comment by heejit** [comment]
I used the styler function multiple times once at the start and another end of the function and it is working is that the right way ?

**Comment by kkoreilly** [comment]
I would suggest adding one Styler that checks a bool condition to determine what styles to set, and then set that bool appropriately before and after the function call.

**Reply by heejit** [reply]
okay that will be better
